docs/architecture/adr/adr-006-in-cluster-db.md

```diff
@@ -0,0 +1,113 @@
+# ADR 006: In-cluster Database
+
+## Context
+
+Microsoft has decided to sunset Azure Database for MariaDB summer 2025.
+They recommend moving to their Azure Database for MySql instead.
+
+The project has experienced project wide down time due to database crashes on
+multiple occasions:
+
+- 17.01.2025: Database went down due to a restart initiated by the Azure Product
+Group team taking unsanctioned action
+- 15.01.2025: Database went down due to a restart initiated by the Azure Product
+Group team taking unsanctioned action
+- 9.01.2025: Database went down as a result of to many login requests
+- 28.11.2024: Database went down due to too many active connection attempts
+- 7.11.2024: Database went down due to too many connections
+- Summer 2024: Database was restarted by Azure for unscheduled and unnotified
+maintenance
+
+Microsofts datbase server logs are prohibitively expensive.
+Microsoft misconfigured the database provision-time.
+Microsofts support is tiered and direct access to technical staff is prevented
+by Azure support policy.
```

Comment:
I think the important point here is that the current support tier is useless, and useful support is prohibitively expensive. It is unclear what the point is from the current sentence.